backend/src/main/java/com/tourneynizer/tourneynizer/model/UserInfo.java

```diff
@@ -0,0 +1,13 @@
+package com.tourneynizer.tourneynizer.model;
+
+public class UserInfo {
+
+    public int wins, losses, tournaments, matches;
+
+    public UserInfo(int wins, int losses, int tournaments, int matches) {
+        this.wins = wins;
+        this.losses = losses;
+        this.tournaments = tournaments;
+        this.matches = matches;
+    }
+}
```

Comment:
Don't you also want to have a connecting user object or id value?

Reply:
User hasa UserInfo

Reply:
Did you add it to the User class yet?